src/main/kotlin/no/nav/familie/ef/sak/opplysninger/personopplysninger/mapper/AdresseHjelper.kt

```diff
@@ -46,5 +46,13 @@ object AdresseHjelper {
         return gjeldende.startdatoForKontrakt.isEqualOrBefore(nå) &&
             (gjeldende.sluttdatoForKontrakt == null || gjeldende.sluttdatoForKontrakt.isEqualOrAfter(nå))
     }
-}
 
+    fun harDeltBosted(barn: BarnMedIdent?, dato: LocalDate): Boolean {
+        if (barn == null) {
+            return false
+        }
+        val gjeldende = barn.deltBosted.gjeldende() ?: return false
+        return gjeldende.startdatoForKontrakt.isEqualOrBefore(dato) &&
+            (gjeldende.sluttdatoForKontrakt == null || gjeldende.sluttdatoForKontrakt.isEqualOrAfter(dato))
+    }
```

Comment:
Er det ryddigere å bruke comparison operators? 
```suggestion
        return gjeldende.startdatoForKontrakt <= dato &&
            (gjeldende.sluttdatoForKontrakt == null || gjeldende.sluttdatoForKontrakt >= dato)
```